modules/adding-a-custom-model-serving-runtime.adoc

```diff
@@ -25,7 +25,11 @@ NOTE: {productname-short} includes the OpenVINO Model Server model-serving runti
 . From the {productname-short} dashboard, click *Settings* > *Serving runtimes*.
 +
 The *Serving runtimes* page opens and shows the model-serving runtimes that are already installed and enabled in your {productname-short} deployment. By default, the OpenVINO Model Server runtime is pre-installed and enabled in {productname-short}.
-. To add a new, custom runtime, click *Add serving runtime*.
+. To add a custom runtime:
+
+** To start with an existing runtime, click the More Options icon (image:images/osd-ellipsis.png[More options]) next to the existing rumtime and then click *Duplicate*. Skip to Step 5. 
+
+** To add a new custom runtime, click *Add serving runtime*.
 +
 The *Add serving runtime* page opens.
```

Comment:
Currently, because of the way "The **Add serving runtime** page opens" line is indented, it looks like it is the result of the second bullet only. We want to indent it to the first sentence of the step instead.

To solve this, I suggest enclosing the whole bullet list in `--`delimiters:

````
. To add a custom runtime, choose one of the following options:
+
--
** To start with an existing runtime, click the action menu (*&#8942;*) next to the existing runtime and then click *Duplicate*. Skip to Step 5. 

** To add a new custom runtime, click *Add serving runtime*.
--
+
The *Add serving runtime* page opens.
````